x/denommetadata/keeper/keeper.go

```diff
@@ -46,7 +47,7 @@ func (k *Keeper) CreateDenomMetadata(ctx sdk.Context, metadata banktypes.Metadat
 
 // UpdateDenomMetadata returns the denommetadata of the specified denom
 func (k *Keeper) UpdateDenomMetadata(ctx sdk.Context, metadata banktypes.Metadata) error {
-	found := k.HasDenomMetadata(ctx, metadata.Base)
+	_, found := k.bankKeeper.GetDenomMetaData(ctx, metadata.Base)
 	if !found {
```

Comment:
why this change?
if denommetadat keeper.HasDenomMetadata changes impl then this will not be consistent